inttest/basic/basic_test.go

```diff
@@ -111,6 +114,19 @@ func (s *BasicSuite) TestK0sGetsUp() {
 		s.Require().NoError(common.VerifyKubeletMetrics(ctx, kc, node))
 	}
 
+	s.T().Log("checking kube-router gobgp functionality")
+	kubeRouterPods, err := kc.CoreV1().Pods("kube-system").List(ctx, metav1.ListOptions{LabelSelector: "k8s-app=kube-router"})
+	s.Require().NoError(err)
+	// Just take the first running pod for execing the gobgp command
+	for _, pod := range kubeRouterPods.Items {
+		if pod.Status.Phase == corev1.PodRunning {
+			out, err := common.PodExecCmdOutput(kc, restConfig, pod.Name, "kube-system", "gobgp global")
+			s.Require().NoError(err)
+			s.Assert().Regexp(`AS:\s+64512`, out)
```

Comment:
What a magic number 😂

What does this mean? Would be awesome if we had some comment about this.

Reply:
That's the default bgp ASN number used by kube-router